app/assets/stylesheets/components/auth-form.scss

```diff
@@ -1,26 +1,27 @@
+@import "base/colors";
 
 .auth-form {
+  background: #f8fbfd;
```

Comment:
Color literals like `#f8fbfd` should only be used in variable declarations; they should be referred to via variable everywhere else.